contracts/SplitMain.sol

```diff
@@ -537,6 +561,15 @@ contract SplitMain is ISplitMain {
     }
   }
 
+  /**
+  * @notice claim yield generated by split main and send it to the yield recipient.
+  * @param _token rebasing token to claim yield from, WETH or USDB.
+  * @param _amount amount of yield to claim.
+  */
+  function claimYield(address _token, uint256 _amount) external {
```

Comment:
do we need this? i'm pretty sure we dont since we set the governor to another address

Reply:
I think this is needed, governor can only claim native yield and fees